web/source/osk/abstractions.ts

```diff
@@ -0,0 +1,9 @@
+namespace com.keyman.osk {
```

Comment:
Can we name this file `keytip.ts` or `keytip.interface.ts` or something like that -- by its content rather than its category? I'd prefer not to have abstractions.ts grow into a grab-bag of various abstractions.

Reply:
If that's your preference, I could do that.  I'd expected that with how small this file would be without any extras, you'd have the opposite preference.  My mistake.

Reply:
😁 

I think the move towards smaller source files is healthy, generally. But more important is single-purpose. `utils.ts` is a problem we are still trying to resolve now!

Reply:
> `utils.ts` is a problem we are still trying to resolve now!

That _is_ true; I've noted the need to split off, at minimum, the `getAbsolute` functions into their own thing.  They're used both for the OSK and for element-attachment.  :+1: